test/fixtures/pages/helpers-demo-collections.hbs

```diff
@@ -0,0 +1,21 @@
+{{#* inline "collection-item"}}
```

Comment:
Nice use of an inline partial!

Not sure it matters, but should there be no space between the `#*` and the `inline`? I was looking at the [Handlebars example](http://handlebarsjs.com/partials.html#inline-partials).